.github/workflows/build.yml

```diff
@@ -463,6 +463,64 @@ jobs:
           name: 'PackSquash CLI AppImage zsync (arm64)'
           path: PackSquash-*-aarch64.AppImage.zsync
 
+  docker-image:
```

Comment:
The job should not work for pull_request. Please add `if` by referring to other jobs.

Reply:
@sya-ri 
We should probably make it build, since at the moment binaries do build at pull requests. But we can make it so it won't be pushed to ghcr.io.
It's done this way in official [build-push-action documentation](https://github.com/docker/build-push-action/blob/master/docs/advanced/tags-labels.md) (check the last step in a job). Is it fine?

Reply:
I went and made it build, but not push on pull requests. Marking this as solved